### PR TITLE
Feature/views

### PR DESCRIPTION
**Pull request by RileyManda**
I have completed the task of adding views to the BlogApp Application according to the stipulated [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md): 🌱 
**NB:To reviewer- Styling has not been implemented to this Milestone: Main area of focus was functionality and view/routes/crud operations as indicated in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)**

# Implementations: 

### Views

> User List
> User Details View
> Post List
> Post Details View

 In addition I have ensured the following general requirements:
 
[Linters](https://github.com/RileyManda/blog-app/actions/workflows/linters.yml)
### General requirements

> Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
> Made sure that I have used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
> Made sure that I have documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
> Followed list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).

Please kindly review my implementation.

Thank you 🙏🏻

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,32 @@
...
+    </div>
+  </div>
+</div>
+<div class="bio">
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+
+<div class="posts">
+  <% @user.posts.order(created_at: :desc).limit(3).each do |post| %>
```
Comment (IngDavidTM) [inline app/views/users/show.html.erb]: - [x] While you did a fantastic job as a project requirement you should use the method #recent_post method to render the user's 3 most recent post 👍

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,43 @@
...
+        <div class="post">
+          <h2><%= post.title %></h2>
+          <p><%= post.text %></p>
+          <div class="comment-likes-container">
+            <p>Comments: <%= post.comments.count %></p>
+            <p>Likes: y</p>
+          </div>
+          <% if post.comments.any? %>
+            <ul>
+              <% post.comments.each do |comment| %>
```
Comment (IngDavidTM) [inline app/views/posts/index.html.erb]: - [ ] Same here you should use the method post.recent_comments to render 5 most recent comments 👍

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @RileyManda 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RileyManda** [comment]
I have completed the task of making iterations to the Posts|Users views by adding the recent posts method as requested: 🌱 


# Implementations: 

### Views

> Used recent posts method in Users view
> Used recent posts method in Posts/index view


 In addition I have ensured the following general requirements:
 
[Linters](https://github.com/RileyManda/blog-app/actions/workflows/linters.yml)
### General requirements

> Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
> Made sure that I have used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
> Made sure that I have documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
> Followed list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).


Please kindly review my implementation.

Thank you 🙏🏻

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @RileyManda ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Added post method for recent comments ✔️
- Added post method for recent posts ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**